app/models/payment_provider_customers/stripe_customer.rb

```diff
@@ -2,12 +2,33 @@
 
 module PaymentProviderCustomers
   class StripeCustomer < BaseCustomer
+    ALLOWED_PAYMENT_METHODS = %w[card sepa_debit].freeze
+
+    validates :provider_payment_methods, presence: true
+    validate :allowed_provider_payment_methods
+
     def payment_method_id
       get_from_settings('payment_method_id')
     end
 
     def payment_method_id=(payment_method_id)
       push_to_settings(key: 'payment_method_id', value: payment_method_id)
     end
+
+    def provider_payment_methods
+      get_from_settings('provider_payment_methods')
+    end
+
+    def provider_payment_methods=(provider_payment_methods)
+      push_to_settings(key: 'provider_payment_methods', value: provider_payment_methods.to_a)
+    end
+
+    private
+
+    def allowed_provider_payment_methods
+      if (provider_payment_methods - ALLOWED_PAYMENT_METHODS).present?
```

Comment:
Style/GuardClause: Use a guard clause (`return unless (provider_payment_methods - ALLOWED_PAYMENT_METHODS).present?`) instead of wrapping the code inside a conditional expression. (https://rubystyle.guide#no-nested-conditionals)